pyvista/core/dataset.py

```diff
@@ -773,36 +1060,129 @@ def copy_meta_from(self, ido: 'DataSet'):
+
+        Access an array from point_arrays
+
+        >>> mesh.point_arrays['my_other_array']
+        pyvista_ndarray([0, 1, 2, 3, 4, 5, 6, 7])
+
+        Or access it directly from the mesh.
+
+        >>> mesh['my_array'].shape
+        (8,)
+
+        """
+        return DataSetAttributes(self.GetPointData(), dataset=self,
+                                 association=FieldAssociation.POINT)
 
     def clear_point_arrays(self):
-        """Remove all point arrays."""
+        """Remove all point arrays.
+
+        Examples
+        --------
+        Clear all point arrays from a mesh.
+
+        >>> import pyvista
+        >>> import numpy as np
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.point_arrays.keys()
+        ['Normals']
+        >>> mesh.clear_point_arrays()
+        >>> mesh.point_arrays.keys()
+        []
+
+        """
         self.point_arrays.clear()
 
     def clear_cell_arrays(self):
         """Remove all cell arrays."""
         self.cell_arrays.clear()
 
     def clear_arrays(self):
-        """Remove all arrays from point/cell/field data."""
+        """Remove all arrays from point/cell/field data.
+
+        Examples
+        --------
+        Clear all arrays from a mesh.
+
+        >>> import pyvista
+        >>> import numpy as np
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.point_arrays.keys()
+        ['Normals']
+        >>> mesh.clear_arrays()
+        >>> mesh.point_arrays.keys()
+        []
+
+        """
         self.clear_point_arrays()
         self.clear_cell_arrays()
         self.clear_field_arrays()
 
     @property
     def cell_arrays(self) -> DataSetAttributes:
         """Return vtkCellData as DataSetAttributes."""
-        return DataSetAttributes(self.GetCellData(), dataset=self, association=FieldAssociation.CELL)
+        return DataSetAttributes(self.GetCellData(), dataset=self,
+                                 association=FieldAssociation.CELL)
 
     @property
     def n_points(self) -> int:
-        """Return the number of points in the entire dataset."""
+        """Return the number of points in the entire dataset.
+
+        Examples
+        --------
+        Create a mesh and return the number of points in the
+        mesh.
+
+        >>> import pyvista
+        >>> cube = pyvista.Cube().clean()
+        >>> cube.n_points
+        8
+
+        """
         return self.GetNumberOfPoints()
 
     @property
     def n_cells(self) -> int:
-        """Return the number of cells in the entire dataset."""
+        """Return the number of cells in the entire dataset.
+
+        Examples
+        --------
+        Create a mesh and return the number of cells in the
+        mesh.
+
+        >>> import pyvista
+        >>> cube = pyvista.Cube().clean()
+        >>> cube.n_cells
+        6
+
+        Notes
+        -----
+        For :class:`pyvista.PolyData`, this will also return the
+        number of faces.
```

Comment:
What does "also" mean here? They are included in the total? And what else is? I thought faces were all that PolyData had.

Reply:
```suggestion
        This is identical to :attr:`n_faces <pyvista.PolyData.n_faces>`
        in :class:`pyvista.PolyData`.
```

Reply:
"also" means I didn't think about the documentation enough :)